lib/src/emoji/widgets/search_bar_widget.dart

```diff
@@ -47,7 +47,7 @@ class _SearchBarWidgetState extends State<SearchBarWidget> {
       constraints: const BoxConstraints(minWidth: 360.0, maxWidth: 400.0),
       controller: searchTextController,
       focusNode: widget.focusNode,
-      hintText: AppLocalizations.of(context)!.searchHintText,
+      hintText: t.searchHintText,
```

Comment:
Mystery / single letter variables can be confusing to people orienting
themselves with the code; can we change the access variable to `translations`
for clarity?

```suggestion
      hintText: translations.searchHintText,
```

Reply:
Yes, this option can be set on build.yml!